### DISCUSSION TITLE
Install insider preview WSA latest on Windows 11 non-insider builds

### DISCUSSION THREAD

**Discussion by joeyoropesa-dev** [LSPosed/MagiskOnWSALocal#641] [Q&A]
How can I install it and is it possible to set custom amount of RAM for example if I have 2GB RAM installed to be able to run WSA just using the custom mb allocation for example, to write 1024 or 1336 or 915 or any other amount that will not eat the whole installed RAM.

I heard that the Microsoft latest (insider build release for WSA) WSA have that feature so I want to test it (try it)

**Comment by Keritial** [comment]
That doesn't seem to be a problem since there's no (probably) restrictions for side-loading. I'm running WSA 2306 on a stable release of Windows 11.

![Windows version](https://github.com/LSPosed/MagiskOnWSALocal/assets/50075277/3c4ac786-a482-4228-8aa3-6344a8d9cd20)
![Windows Subsystem for Android version](https://github.com/LSPosed/MagiskOnWSALocal/assets/50075277/61ddf2d0-4bdc-41ea-800e-5a945a46366f)

Just follow the guide in the project README file, or directly install the .AppxBundle package with App Installer if you don't need root or Gapps.

You can get a download link of WSA package at https://store.rg-adguard.net. (Be careful since it is a 3rd-party website I just randomly Googled, but the links it generates are from Microsoft.) And the store page of WSA is https://apps.microsoft.com/store/detail/9P3395VX91NR.

![App Installer package information page with WSA already installed](https://github.com/LSPosed/MagiskOnWSALocal/assets/50075277/df7e8be7-083c-4d53-8a8b-55515063c16f)

You can set a custom memory allocation size, however, the minimal value is 2GB.

![Memory Allocation](https://github.com/LSPosed/MagiskOnWSALocal/assets/50075277/fbc1a725-261c-491e-857a-711eacf5c9b5)

**Reply by joeyoropesa-dev** [reply]
Wow... my PC is literally 2GB of Physical RAM installed in it. Anyway, when I start the tool and select custom (MB) option in Memory allocation that says what amount I can put between 19** gb and 2 gb, nothing works. Button is unclickable and greyed out and any other of options to use like low or high doesn't do anything - only staying in default option that is even higher than my physical RAM for 1GB more.

Is that normal due to the Android 13 minimum RAM requirements by system or that's an bug or the tool is checking for compatibility so that if I'm not compatible with the WSA, it will automatically disable those options.


Anyway, I would love to use registry editor to apply those changes if the app doesn't allow me in GUI way - is there anyone who knows how WSA Settings works and where those settings can be found in registry editor to find the correct flags I need to change to apply different configuration for the default profile (RAM usage)
